.travis.yml

```diff
@@ -2,7 +2,8 @@ language: node_js
 sudo: false
 addons:
   code_climate:
-    repo_token: 36766266e0f1834997de90e358ea07873eccb07006f9efb7369c35823024481c
+    repo_token:
```

Comment:
I don't feel 100% comfortable keeping this public, but I realize we don't consider the repo token private data. While you can't fetch anything sensitive using the token, you can still disrupt repo test coverage data.

I changed this to a plaintext update for now.